index.html

```diff
@@ -245,7 +252,6 @@ <h3 class="project-title">Professional Art Printing Data</h3>
                         </span>
                     </a>
                 </li>
-                <li class="no-project"></li>
             </ul>
         </section>
         <section class="about-me decorative-background2">
```

Comment:
- [ ] Please make sure the about section background image is vertically centered as on the design. For that, you need to reduce the width of about section details.

### Your design

![image](https://user-images.githubusercontent.com/83803414/160230029-0f194bad-f50e-4bd0-a1cf-8750893eb6c5.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/160230041-58f9d003-e42b-49be-ba53-c3f46da51645.png)